src/fidesops/schemas/saas/saas_config.py

```diff
@@ -194,6 +211,17 @@ class SaaSConfig(BaseModel):
     test_request: SaaSRequest
     data_protection_request: Optional[SaaSRequest] = None  # GDPR Delete
 
+    @validator("type", pre=True)
+    def validate_log_level(cls, value: str) -> str:
```

Comment:
Copy/paste issue. I'd rename this to something like `validate_saas_type` or similar